src/sentry/utils/suspect_resolutions/resolved_in_active_release.py

```diff
@@ -31,4 +30,4 @@ def is_resolved_issue_within_active_release(issue: Group) -> bool:
 
     now_minus_1_hour = timezone.now() - timedelta(hours=1.0)
 
-    return now_minus_1_hour <= latest_deploy_release.date_finished <= timezone.now()
+    return bool(now_minus_1_hour <= latest_deploy_release.date_finished <= timezone.now())
```

Comment:
Why do we need to cast this as a boolean - isn't the result of the expression already a boolean?

Reply:
It has to do with typing. I added a couple of our source files to mypy. And now it complains that this expression returns `Any`. Since we know this expression should return a bool, the casting is really there for the type checking.